Source/SourceKittenFramework/Request.swift

```diff
@@ -340,3 +340,70 @@ extension Request: CustomStringConvertible {
     /// A textual representation of `Request`.
     public var description: String { return String(UTF8String: sourcekitd_request_description_copy(sourcekitObject))! }
 }
+
+private func interfaceForModule(module: String) -> [String: SourceKitRepresentable] {
+    let xctestConfigurationPath = NSProcessInfo.processInfo().environment["XCTestConfigurationFilePath"]!
+    let buildPath = xctestConfigurationPath.substringToIndex(xctestConfigurationPath.rangeOfString("/Intermediates/")!.startIndex)
```

Comment:
Never mind, I think I fixed it by passing the compiler arguments from SourceKittenFramework to the interface calls.